tests.py

```diff
@@ -35,19 +40,34 @@ def test_editor_loads_native():
         assert extensions["cmtype"] == viscm.cmtype
 
         colors = data["colors"]
-        colors = [[int(c[i:i + 2], 16) / 256 for i in range(0, 6, 2)] for c in [colors[i:i + 6] for i in range(0, len(colors), 6)]]
+        colors = [
+            [int(c[i : i + 2], 16) / 256 for i in range(0, 6, 2)]
+            for c in [colors[i : i + 6] for i in range(0, len(colors), 6)]
+        ]
         editor_colors = viscm.cmap_model.get_sRGB(num=256)[0].tolist()
         for i in range(len(colors)):
             for z in range(3):
                 assert colors[i][z] == np.rint(editor_colors[i][z] / 256)
 
+
+# import matplotlib as mpl
+# from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
+# from matplotlib.backends.qt_compat import QtCore, QtGui
+#
 # def test_editor_add_point():
```

Comment:
Pulled all the imports associated with this commented block into the commented block. Any reason to keep this function though?